tlatools/org.lamport.tlatools/test-model/ParameterizedInstantiation/ParamInstance.tla

```diff
@@ -0,0 +1,31 @@
+--------------------------- MODULE ParamInstance ---------------------------
+EXTENDS Naturals
+
+VARIABLE value
+
+  ------ MODULE Ring ------
+  VARIABLE cardinality
+  
+  Init == /\ cardinality > 1
+          /\ value \in (0 .. cardinality)
```

Comment:
ya ya.. should be 0..(cardinality - 1) -- doesn't matter in the larger scope of things.